modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/NewQuestion.es.js

```diff
@@ -142,7 +145,7 @@ export default withRouter(
 						siteKey: context.siteKey,
 					},
 				})
-					.then(debounceCallback)
+					.then(({error}) => processResponse(error))
```

Comment:
this looks like it's capturing the error and doing sth with it instead of processing the error or the result... it's Ok, just a comment.